tests/baselines/reference/jsObjectsMarkedAsOpenEnded.types

```diff
@@ -1,12 +1,12 @@
 === tests/cases/conformance/salsa/a.js ===
 var variable = {};
->variable : { a: number; }
+>variable : typeof variable

```

Comment:
Do we want to consider not using `typeof` on the printout of expando types if they don't contain call/construct signatures? Just a thought.

Reply:
Given that this is a JS scenario, I think that the "value name as type name" pattern is actually more understandable than a structural type -- for authors, though not for us. chrome-devtools-frontend, for example: would they rather see `typeof Common` or `{ Ns1: { ... }, Ns2: { ... } }`?